calico_node/Makefile

```diff
@@ -1,60 +1,31 @@
 ###############################################################################
 
-GO_BUILD_VER?=v0.7
+GO_BUILD_VER?=v0.8
 CALICO_BUILD?=calico/go-build:$(GO_BUILD_VER)
 
 CALICO_NODE_DIR=$(dir $(realpath $(lastword $(MAKEFILE_LIST))))
 VERSIONS_FILE?=$(CALICO_NODE_DIR)/../_data/versions.yml
 
+# Read current stream version from _data/versions.yml
+RELEASE_STREAM?=$(shell cat $(VERSIONS_FILE) | $(YAML_CMD) read - "currentReleaseStream")
+
 ###############################################################################
 # Determine whether there's a local yaml installed or use dockerized version.
 # Note in order to install local (faster) yaml: "go get github.com/mikefarah/yaml"
 YAML_CMD:=$(shell which yaml || echo docker run --rm -i $(CALICO_BUILD) yaml)
 
-# Read current stream version from _data/versions.yml
```

Comment:
I'm not sure these changes should be part of the release ... @neiljerram and I have been discussing some changes to the version reporting, but should probably be a separate commit (unless Neil is approving this).

Reply:
This was an accidental revert that happened when I reverted another PR.  Fixed up.